app.json

```diff
@@ -20,7 +20,8 @@
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "package": "com.frankreed.GolfApp"
```

Comment:
What's this 👀

Reply:
I've been building this app into an android apk and this is for that